packages/use-contractkit/package.json

```diff
@@ -41,6 +42,7 @@
   "devDependencies": {
     "@types/isomorphic-fetch": "^0.0.35",
     "@types/ledgerhq__hw-transport-webusb": "^4.70.1",
+    "@types/qrcode": "^1.4.2",
     "@types/qrcode.react": "^1.0.2",
```

Comment:
we should remove this tyep right? since `qrcode.react` is gone

Reply:
Good catch!